docs/scripts/api/buildInterfacesDocumentation.ts

```diff
@@ -356,26 +292,91 @@ export default async function buildInterfacesDocumentation(
       // eslint-disable-next-line no-console
       console.log('Built JSON file for', parsedInterface.name);
     } else {
+      const content = {
+        name: parsedInterface.name,
+        imports: generateImportStatement(parsedInterface, projects),
+        ...extractDemos(parsedInterface.tags.demos),
+        properties: {},
+      };
+
+      const translations = {
+        interfaceDescription: renderMarkdown(
+          escapeCell(linkify(parsedInterface.description, documentedInterfaces, 'html')),
```

Comment:
```suggestion
          linkify(parsedInterface.description, documentedInterfaces, 'html'),
```

<img width="676" src="https://github.com/mui/mui-x/assets/13808724/6f53a2ef-aada-4e06-a930-26a5937916de">